src/components/layout/navbar-profile-menu.tsx

```diff
@@ -1,15 +1,48 @@
 'use client'
 
-import { AppShellSection, NavLink } from '@mantine/core'
+import { AppShellSection, Collapse, NavLink } from '@mantine/core'
 import { UserAvatar } from '@/components/user-avatar'
 import { UserName } from '@/components/user-name'
-import { CaretRight } from '@phosphor-icons/react/dist/ssr'
+import { CaretRight, Lock, SignOut, User } from '@phosphor-icons/react/dist/ssr'
 import styles from './navbar-items.module.css'
+import { useDisclosure } from '@mantine/hooks'
+import { useClerk } from '@clerk/nextjs'
 
 export function NavbarProfileMenu() {
+    const { signOut, openUserProfile } = useClerk()
+    const [opened, { toggle }] = useDisclosure()
+
     return (
         <AppShellSection>
-            {/* Placeholder for collapsable menu */}
+            <Collapse in={opened} bg={'var(--mantine-color-purple-9)'}>
```

Comment:
sharing some knowledge around [mantine colors](https://mantine.dev/theming/colors/#colors-index-reference), any mantine component can take a simple `purple.9` or `blue.4`

`c` is typically the font color
`color` usually controls the whole internal mantine theming stuff

but you can use it for any of the props such as bg too i believe 😄